test/release/examples/primers/FFTWlib.chpl

```diff
@@ -195,73 +195,73 @@ proc runtest(param ndim : int, fn : string) {
       // Define domains to extract the real and imaginary parts for
       // in-place transforms
       //
-      reD = rD[..,..,0..(2*ldim-1) by 2]; 
-      imD = rD[..,..,1..(2*ldim-1) by 2]; 
+      reD = rD[..,..,0..(2*ldim-1) by 2];
+      imD = rD[..,..,1..(2*ldim-1) by 2];
     }
   }
 
   // FFTW does not normalize inverse transforms, so just compute the
   // normalization constant.
 
-  var norm = * reduce dims;
+  var norm =   reduce dims;
```

Comment:
The intention of my `sed` command was to remove cases like [these](https://github.com/chapel-lang/chapel/pull/4524/commits/acab97ccb943de8e3be8d25e8f2b55aa21e55c92#diff-31d47e363ffa2ec03395b55a87a4c4b4L2):

```chapel
/*
 *  file.chpl
 *
 *  comments
*/
```

However, my regex was poorly defined, and I didn't observe the havoc it caused until several commits later. Fix on the way.